test/FunctionalTests/Tests/DataServices/UnitTests/Client.TDD.Tests/Tests/DateTimeClientSupportTest.cs

```diff
@@ -0,0 +1,79 @@
+﻿namespace Microsoft.OData.Client.TDDUnitTests.Tests
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Collections.ObjectModel;
+    using System.Linq;
+    using Microsoft.OData.Client;
+    using Microsoft.VisualStudio.TestTools.UnitTesting;
+    using System.Globalization;
+    using Microsoft.Test.OData.Services.TestServices;
+    using pkr = Microsoft.Test.OData.Services.TestServices.DateTimePrimitiveTypeReference;
+
+    [TestClass]
+    public class DateTimeClientSupportTest

```

Comment:
Besides these test cases, we'd better and some other test cases.
1. Query a single DateTime Property.
2. Use DateTime as a function parameter.
3. Use DateTime an action parameter.
4. Post/Update an entity with a DateTime property.
5. A function or action returns a DateTime.